src/lectern-client/schema-entities.ts

```diff
@@ -3,14 +3,14 @@ import { DeepReadonly } from 'deep-freeze';
 const L = loggerFor(__filename);
 
 export class DataRecord {
-  readonly [k: string]: string;
+  readonly [k: string]: string | string[];
 }
 
 export class TypedDataRecord {
   readonly [k: string]: SchemaTypes;
 }
 
-export type SchemaTypes = string | boolean | number | undefined;
+export type SchemaTypes = string | string[] | boolean | boolean[] | number | number[] | undefined;
```

Comment:
how can boolean be an array ?

Reply:
It could be if we set `isArray: true` in the schema, which is set at the field level. There is nothing in the schema that stops one from doing that. I agree it probably won't make sense in any of our contexts, but I just kept it general with the types since lectern client doesn't have any concepts of whether the input makes sense in the context.

Reply:
yeah I saw that in lectern.. so it's a feature ['true', 'Yes', 'Y', 'Si', 'Oui']

Reply:
Yea I guess. Lol could use it for psuedo binary inputs too eh [true, false, true, false] = [10101]